source/common/formatter/substitution_formatter.cc

```diff
@@ -1078,7 +1078,7 @@ const StreamInfoFormatter::FieldExtractorLookupTbl& StreamInfoFormatter::getKnow
                             [](const std::string&, const absl::optional<size_t>&) {
                               return std::make_unique<StreamInfoDurationFieldExtractor>(
                                   [](const StreamInfo::StreamInfo& stream_info) {
-                                    return stream_info.requestComplete();
+                                    return stream_info.currentDuration();
```

Comment:
I think we need another way to mark mid-stream access logs before we merge this. Otherwise, the release note https://github.com/envoyproxy/envoy/pull/25824 is no longer true - we can't tell if the access log is final or intermediate.

Reply:
Yeah I agree on it. I'll create a PR to add "StreamState" enum to stream info or something similar.